time|level|timeoffset|message|location
071015_171616.634|INFO|000.000392| Software version: Plus-2.3.0.4217 - Linux| in vtkPlusLogger(183)
071015_171616.634|INFO|000.000459| Logging at level 3 (INFO) to file: /home/snr/Support/Plus-build-codeblocks/bin/Output/071015_171616_PlusLog.txt| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusServer/Testing/PlusServer.cxx(85)
071015_171616.636|INFO|000.002294| Selected US image orientation: MF| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkPlusDataSource.cxx(264)
071015_171616.636|INFO|000.002535| VideoDevice: Local time offset: 0ms| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkPlusDevice.cxx(869)
071015_171616.639|INFO|000.005393| TrackerDeviceSavedDataset: Local time offset: 0ms| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkPlusDevice.cxx(869)
071015_171616.639|ERROR|000.005502| Image with relative path 'PivotCalibration.mha' cannot be found neither relative to image directory (/home/snr/Support/Plus-build-codeblocks/PlusLib/data/TestImages) nor to device set configuration directory (/home/snr/Documents/slicerigt-data/SlicerIGT-Data/Config_roos) nor the current working directory directory (/home/snr/Support/Plus-build-codeblocks/bin)| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusCommon/vtkPlusConfig.cxx(829)
071015_171616.639|ERROR|000.005597| Image with relative path 'PivotCalibration.mha' cannot be found neither relative to image directory (/home/snr/Support/Plus-build-codeblocks/PlusLib/data/TestImages) nor to device set configuration directory (/home/snr/Documents/slicerigt-data/SlicerIGT-Data/Config_roos) nor the current working directory directory (/home/snr/Support/Plus-build-codeblocks/bin)| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusCommon/vtkPlusConfig.cxx(829)
071015_171616.639|ERROR|000.005613| Unable to locate file: PivotCalibration.mha. Please verify location on disk.| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/SavedDataSource/vtkSavedDataSource.cxx(599)
071015_171616.639|ERROR|000.005627| Failed to read parameters of device: TrackerDeviceSavedDataset (type: SavedDataSource)| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/DataCollection/vtkDataCollector.cxx(127)
071015_171616.639|ERROR|000.005664| Datacollector failed to read configuration| in /home/snr/Support/Plus-build-codeblocks/PlusLib/src/PlusServer/Testing/PlusServer.cxx(117)
